src/api/models/DeliveryAccount.ts

```diff
@@ -6,7 +6,6 @@ export type DeliveryAccount = {
     readonly id: number;
     type: 'hellosign' | 'aws_s3';
     readonly sub_type: string;
-    readonly used_by: number;
```

Comment:
This shouldn't have happened.

Reply:
Updated in commit 5125a9bf9687065f8ca3d3e239d7be04609ab1d4